### DISCUSSION TITLE
New notification feature for completed set requests

### DISCUSSION THREAD

**Discussion by Tsearo** [RetroAchievements/RAWeb#798] [Ideas]
Some sort of opt in notifications, email or site message, would be nice when a set that you have a game request is finished. Particularly for users that aren't on discord or users that don't check for updates frequently.

**Comment by meleu** [comment]
Unfortunately there's no way to trigger the "achievement set has been finished" condition.

Probably something for v2.

**Comment by Jamiras** [comment]
I believe this is handled by the claims feature (#1026)
